server.js

```diff
@@ -33,7 +33,7 @@ var _ = require('underscore'),
     monthlyRegistrations = require('./controllers/monthly-registrations'),
     monthlyDeliveries = require('./controllers/monthly-deliveries'),
     exportData = require('./controllers/export-data'),
-    messages = require('./controllers/messages'),
+    messages = require('./controllers/message-utils'),
```

Comment:
Yeah I planned on renaming it, I just never got around to it I guess. I think I didn't want it in the same commit. I'll rename it now.